### PR TITLE
chore: add store-identity protocol, bump deps.

### LINKED ISSUE #20: Add new store-identity protocol.

### PR DESCRIPTION
**Pull request by whilo**
Fixes #20.

### CODE REVIEW

File: src/datahike_jdbc/core.clj [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
-  (:require [datahike.store :refer [empty-store delete-store connect-store default-config config-spec release-store]]
+  (:require [datahike.store :refer [empty-store delete-store connect-store default-config config-spec release-store store-identity]]
             [datahike.config :refer [map-from-env]]
             [konserve-jdbc.core :as k]
             [clojure.spec.alpha :as s]))
 
+(defmethod store-identity :jdbc [store-config]
+  (let [{:keys [jdbcUrl dbtype host port dbname]} store-config]
+    (if jdbcUrl
+      [:jdbc jdbcUrl]
```
Comment (TimoKramer) [inline src/datahike_jdbc/core.clj:10]: You wanna mention it in the README?

**Review by TimoKramer — APPROVED** [review]

File: src/datahike_jdbc/core.clj (hunk 1 above)
Comment (whilo) [reply src/datahike_jdbc/core.clj:10]: This is internal, it is just providing a vector that uniquely determines the underlying store for a database. Maybe I am missing where to put it in the README.

Comment (TimoKramer) [reply src/datahike_jdbc/core.clj:10]: ok, got it

**Review by TimoKramer — APPROVED** [review]

**Comment by whilo** [comment]
@TimoKramer The CI/CD pipeline did not run through. I have merged this for now anyway, because otherwise downstream users will not be able to use recent Datahike versions with JDBC.

**Pull request merged**